### ISSUE TITLE
关于参数temperature

### ISSUE THREAD

**Issue by runwean** [thuml/Anomaly-Transformer#12]
您好，非常优秀的文章👍。不过有一个小问题想请教一下作者，在test计算series_loss与prior_loss时，引入了一个参数temperature，请问这个参数的用意是？能否用1/len(prior)代替呢？

-                     series_loss += my_kl_loss(series[u], (
-                             prior[u] / torch.unsqueeze(torch.sum(prior[u], dim=-1), dim=-1).repeat(1, 1, 1,
-                                                                                                    self.win_size)).detach()) * temperature
-                     prior_loss += my_kl_loss(
-                         (prior[u] / torch.unsqueeze(torch.sum(prior[u], dim=-1), dim=-1).repeat(1, 1, 1,
-                                                                                                 self.win_size)),
-                         series[u].detach()) * temperature

**Comment by wuhaixu2016** [comment]
Sure, you can try this setting.
